frontend/src/components/Footer/footer.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../styles/components/variables";
+
+#BcFooter {

```

Comment:
We might want to chat about naming conventions in css, I suspect this is the right way for react, but i suspect this is maybe not known.